tests/benchmarks/boot_time/src/main.c

```diff
@@ -45,37 +35,29 @@ void main(void)
 
 	int freq = sys_clock_hw_cycles_per_sec() / 1000000;
 
-	_start_us  =  __start_time_stamp / freq;
-	s_main_time_stamp =  __main_time_stamp - __start_time_stamp;
-	main_us    =  s_main_time_stamp / freq;
-	s_task_time_stamp =  task_time_stamp   - __start_time_stamp;
-	task_us    =  s_task_time_stamp / freq;
-	s_idle_time_stamp =  __idle_time_stamp - __start_time_stamp;
-	idle_us    =  s_idle_time_stamp / freq;
+	main_us = __main_time_stamp / freq;
+	task_us = task_time_stamp / freq;
+	idle_us = __idle_time_stamp / freq;
 
 	/* Indicate start for sanity test suite */
 	TC_START("Boot Time Measurement");
 
 	/* Only print lower 32bit of time result */
 	TC_PRINT("Boot Result: Clock Frequency: %d MHz\n",
 		 freq);
-	TC_PRINT("__start       : %u cycles, %u us\n",
-		 (u32_t)(__start_time_stamp & 0xFFFFFFFFULL),
-		 (u32_t) (_start_us  & 0xFFFFFFFFULL));
 	TC_PRINT("_start->main(): %u cycles, %u us\n",
-		 (u32_t)(s_main_time_stamp & 0xFFFFFFFFULL),
-		 (u32_t)  (main_us  & 0xFFFFFFFFULL));
+		 __main_time_stamp,
```

Comment:
here as well

Reply:
Not sure what you were seeing here -- if it was an artifact of github (I've seen that before) or a genuine ugly -- but I've reformatted with a slightly different style.